libc-bottom-half/crt/reactor-crt1.c

```diff
@@ -0,0 +1,6 @@
+extern void __wasm_call_ctors(void);
+
+void _activate(void) {
```

Comment:
Should we add `attribute(export-name("_activate"))` so that its is exported from the wasm module without additional linker flags?

Reply:
I guess that `--entry=_activate` is an alternative to this.    If we use the attribute we could pass `--no-entry` for reactors.  Not sure that is preferable...but I do like that fact the using that attributing means that simply including this version crt1.o will cause the export to exist.

Reply:
Ooh. I'd been expecting to submit clang and wasm-ld patches to make them do the right thing when -mexec-model=reactor is given, however using `export_name` is an interesting idea that potentially simplifies things, and more precisely describes what's going on. I've now added a patch which does that.